src/Orchard/Reports/Report.cs

```diff
@@ -7,7 +7,7 @@ public class Report {
             Entries = new List<ReportEntry>();
         }
 
-        public IList<ReportEntry> Entries { get; set;}
+        public List<ReportEntry> Entries { get; set;}
```

Comment:
That must have been something else, e.g. assigning an Array instead of a List. Regardless, the return type should be IList (but initialized to an instance of List as you did). As to why using IList vs List as the type, here's an interesting SO post: http://stackoverflow.com/questions/8717582/why-use-ilist-or-list